docs/05_operations/13_project-structure/01_introduction.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Introduction'
+sidebar_label: 'Introduction'
+sidebar_position: 1
+id: introduction
+keywords: [operations, project structure, introduction]
+tags:
+    - operations
+    - project structure
+    - introduction
+---
+
+This section will explain the Genesis Gradle Settings plugin and the Simplified project structure. The non-simplified project structure is referred to as the Legacy structure.
```

Comment:
I think we could do with more of an explanation here. We should introduce the legacy and simplified structure here. We should explain the benefits of the simplified structure and the reasons for a developer to move to that structure.